x-pack/legacy/plugins/siem/public/components/charts/areachart.tsx

```diff
@@ -59,20 +60,24 @@ const getSeriesLineStyle = (color: string | undefined) => {
 
 // https://ela.st/multi-areaseries
 export const AreaChartBaseComponent = React.memo<{
-  data: ChartConfigsData[];
+  data: ChartSeriesData[];
   width: number | null | undefined;
   height: number | null | undefined;
-  configs?: ChartSeriesConfigs | undefined;
+  configs?: ChartConfigs | undefined;
 }>(({ data, ...chartConfigs }) => {
   const xTickFormatter = get('configs.axis.xTickFormatter', chartConfigs);
   const yTickFormatter = get('configs.axis.yTickFormatter', chartConfigs);
   const xAxisId = getAxisId(`group-${data[0].key}-x`);
   const yAxisId = getAxisId(`group-${data[0].key}-y`);
 
+  const settings = {
+    ...chartDefaultSettings,
+    ...get('configs.settings', chartConfigs),
+  };
   return chartConfigs.width && chartConfigs.height ? (
     <div style={{ height: chartConfigs.height, width: chartConfigs.width, position: 'relative' }}>
       <Chart>
-        <Settings theme={getTheme()} />
+        <Settings {...settings} theme={getTheme()} />
```

Comment:
Taking all the settings from `props.configs` to reserve a path for [onBrushEnd](https://elastic.github.io/elastic-charts/?selectedKind=Interactions&selectedStory=brush%20selection%20tool%20on%20time%20charts&full=0&addons=1&stories=1&panelRight=1&addonPanel=storybook%2Fstories%2Fstories-panel)